src/foam/nanos/notification/NotificationRowView.js

```diff
@@ -0,0 +1,236 @@
+      ^ .dot{
+        height: 500px;
+        width: 500px;
+        border-radius: 50%;
+        color:black;
+      }
+      ^ .net-nanopay-ui-ActionView-optionsDropDown:hover {
+        background-color: rgba(164, 179, 184, 0.3);
+      }
+      ^ .popUpDropDown > div {
+        width: 165px;
+        height: 30px;
+        font-size: 14px;
+        font-weight: 300;
+        letter-spacing: 0.2px;
+        color: #093649;
+        line-height: 30px;
+      }
+      ^ .divBody {
+        background: black !important;
+      }
+      ^ .popUpDropDown > div:hover {
+        background-color: #59a5d5;
+        color: white;
+        cursor: pointer;
+      }
+      ^ .net-nanopay-ui-ActionView-link {
+        background-color: rgba(201, 76, 76, 0.0);
+        width: 135px;
+        height: 40px;
+        border-radius: 2px;
+        border: solid 1px #59a5d5;
+        margin-bottom:20px;
+        margin-left:20px;
+        margin-top:10px;
+      }
+      ^ .net-nanopay-ui-ActionView-link > span {
+        height: 40px;
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: 300;
+        line-height: 2.86;
+        letter-spacing: 0.2px;
+        text-align: center;
+        color: #59a5d5;
+      }
+      ^ .net-nanopay-ui-ActionView-popup {
+        background-color: rgba(201, 76, 76, 0.0);
+        width: 135px;
+        height: 40px;
+        border-radius: 2px;
+        border: solid 1px #59a5d5;
+        margin-bottom:20px;
+        margin-left:20px;
+        margin-top:10px;
+      }
+      ^ .net-nanopay-ui-ActionView-popup > span {
+        height: 40px;
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: 300;
+        line-height: 2.86;
+        letter-spacing: 0.2px;
+        text-align: center;
+        color: #59a5d5;
+      }
+    `,
+
+    properties: [
+      'of',
+      'optionsBtn_',
+      'optionPopup_'
+    ],
+
+    methods: [
+      function initE() { 
+        if ( ! this.data.read ) {
+          this.start('div').addClass('divBodyUnread')
+        } else {
+          this.start('div').addClass('divBody')
+        }
+        this
+        .on('mouseover', this.read)
+        .addClass(this.myClass())
+        this.start(this.OPTIONS_DROP_DOWN, { icon: 'images/ic-options.png', showLabel:true }, this.optionsBtn_$).end()
+        this.add(this.NotificationView.create({of: this.data.cls_, data: this.data}))
+      }
+    ],
+
+    actions: [
+    
+      {
+        name: 'optionsDropDown',
+        label: '',
+        code: function (X) {
+          var self = this;
+          self.optionPopup_ = this.PopupView.create({
+            width: 165,
+            x: -137,
+            y: 40
+          })
+
+          self.optionPopup_.addClass('popUpDropDown')
+            .start('div').add('Remove')
+              .on('click', this.removeNotification)
+            .end()
+            .callIf(this.data.notificationType != 'General', function(){
+              this.start('div')
```

Comment:
Could this be made into an action instead?

Reply:
it could